superflore/parser.py

```diff
@@ -59,4 +59,10 @@ def get_parser(tool_tip, is_generator=True):
             help='location of the upstream repository',
             type=str
         )
+        parser.add_argument(
+            '--skip-keys',
+            nargs='+',
+            help='The specified rosdep keys will be ignored. '
```

Comment:
I don't think "rosdep keys" is the correct term. I think this should read `'packages to skip during regeneration'`. If you intend to skip any packages depending on a certain `rosdep` key, I think that is already handled (at least it is on Gentoo).

Reply:
Done